api/admin/routes.py

```diff
@@ -367,10 +378,7 @@ def admin_auth_service(protocol):
 @requires_admin
 @requires_csrf_token
 def individual_admins():
-    if flask.request.method == 'GET':
-        return app.manager.admin_individual_admin_settings_controller.process_get()
-    else:
-        return app.manager.admin_individual_admin_settings_controller.process_post()
+    return app.manager.admin_individual_admin_settings_controller.process_individual_admins()
```

Comment:
I couldn't find an existing pattern or a way to update `flask.request.method` to be either GET or POST. It seems other tests don't care about it at this point and just test that the main function was called and that the appropriate methods are allowed. So, I switched it to the that pattern here.

Reply:
Yeah, putting the code inside the controller is better.